frontend/src/app/components/containers/complaints/details/caller-information.tsx

```diff
@@ -3,18 +3,33 @@ import { useAppSelector } from "../../../../hooks/hooks";
 import { selectComplaintCallerInformation } from "../../../../store/reducers/complaints";
 import { formatPhoneNumber } from "react-phone-number-input/input";
 import { Card } from "react-bootstrap";
+import { FEATURE_TYPES } from "../../../../constants/feature-flag-types";
+import { isFeatureActive } from "../../../../store/reducers/app";
 
 export const CallerInformation: FC = () => {
-  const { name, primaryPhone, secondaryPhone, alternatePhone, address, email, reportedByCode } = useAppSelector(
-    selectComplaintCallerInformation,
-  );
+  const { name, primaryPhone, secondaryPhone, alternatePhone, address, email, reportedByCode, privacyRequest } =
+    useAppSelector(selectComplaintCallerInformation);
+  const enablePrivacyFeature = useAppSelector(isFeatureActive(FEATURE_TYPES.PRIV_REQ));
+
+  let privacy = "";
+  if (privacyRequest === "Y") {
+    privacy = "Yes";
+  } else if (privacyRequest === "N") {
+    privacy = "No";
+  }
```

Comment:
Isn't this a duplication of something already being done in the automapper?

edit: and also again in the Reducer...

Reply:
No, this isn't duplicating anything from the automapper. Whats happening the automapper is I'm creating a privacy property to handle our carbone workaround without messing up the complaint

Reply:
Understood, 3 places for 3 different things:
1. Report Generation
2. Edit Screen drop downs
3. View Screen display